Academy form model

EliandDeveloper opened the pull request:
Creación del modelo para el formulario de academia con campos básicos.

EliandDeveloper committed:
Academy form model

fow903 commented on addons/academy/__manifest__.py:
```diff
@@ -0,0 +1,35 @@
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
```
Agregale aqui una descripcion

EliandDeveloper replied:
Entendido.

EliandDeveloper committed:
Update in the __manifest__.py summary and description

Updated the previous summary and the description, so we can know what is the purpose of the module.

EliandDeveloper commented on addons/academy/__manifest__.py:
```diff
@@ -0,0 +1,34 @@
+{
+    'name': "Academy",
+
+    'summary': """
+        Este modulo esta dedicado al seguimiento de las informaciones academicas.""",
+
+    'description': """
+        Con este modulo podras darle seguimiento a las informaciones de vital importancia en el entorno academico, del lado de la gestión.
+    """,
+
+    'author': "Elian",
```
Summary, description y author actualizados.
